diff -Naur xine-lib-1.2.4.orig/src/video_dec/libmpeg2/libmpeg2_accel.c xine-lib-1.2.4/src/video_dec/libmpeg2/libmpeg2_accel.c
--- xine-lib-1.2.4.orig/src/video_dec/libmpeg2/libmpeg2_accel.c	2012-02-05 20:17:03.000000000 +0100
+++ xine-lib-1.2.4/src/video_dec/libmpeg2/libmpeg2_accel.c	2013-09-24 21:10:42.580402867 +0200
@@ -17,7 +17,7 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
 #include <xine/xine_internal.h>
diff -Naur xine-lib-1.2.4.orig/src/video_dec/libmpeg2/libmpeg2_accel.h xine-lib-1.2.4/src/video_dec/libmpeg2/libmpeg2_accel.h
--- xine-lib-1.2.4.orig/src/video_dec/libmpeg2/libmpeg2_accel.h	2012-02-05 20:17:03.000000000 +0100
+++ xine-lib-1.2.4/src/video_dec/libmpeg2/libmpeg2_accel.h	2013-09-24 21:08:26.626772723 +0200
@@ -17,7 +17,7 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
 #ifndef LIBMPEG2_ACCEL_H
diff -Naur xine-lib-1.2.4.orig/src/video_dec/libmpeg2/slice_xvmc_vld.c xine-lib-1.2.4/src/video_dec/libmpeg2/slice_xvmc_vld.c
--- xine-lib-1.2.4.orig/src/video_dec/libmpeg2/slice_xvmc_vld.c	2012-02-05 20:17:03.000000000 +0100
+++ xine-lib-1.2.4/src/video_dec/libmpeg2/slice_xvmc_vld.c	2013-09-24 21:10:32.876357267 +0200
@@ -10,11 +10,9 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
  * See the GNU General Public License for more details.
  *
- * You should have received a copy of the GNU General Public License along with
- * this program; if not, write to the Free Software Foundation, Inc.,
- * 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
- *
- *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
 #ifdef HAVE_CONFIG_H
diff -Naur xine-lib-1.2.4.orig/src/video_dec/libmpeg2/xvmc_vld.h xine-lib-1.2.4/src/video_dec/libmpeg2/xvmc_vld.h
--- xine-lib-1.2.4.orig/src/video_dec/libmpeg2/xvmc_vld.h	2012-02-05 20:17:03.000000000 +0100
+++ xine-lib-1.2.4/src/video_dec/libmpeg2/xvmc_vld.h	2013-09-24 21:11:10.290533673 +0200
@@ -10,11 +10,9 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
  * See the GNU General Public License for more details.
  *
- * You should have received a copy of the GNU General Public License along with
- * this program; if not, write to the Free Software Foundation, Inc.,
- * 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
- *
- *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
 #ifndef _XVMC_VLD_H
